budget/index.html

```diff
@@ -0,0 +1,229 @@
+					<article class="ctabox__cta" data-js-ctabox-item>
+						<a href="https://www.reddit.com/r/dashpay/" target="_blank" class="ctabox__link">
+							<div class="ctabox__imagewrapper">
+								<img src="/assets/img/community/reddit_logo.png" class="ctabox__image" >
+							</div>
+							<div class="ctabox__text">
+								<h4 class="ctabox__title">DASHCENTRAL</h4>
+								<p>
+									Budget Monitoring
+									<br>Create Proposal
+									<br> Monitoring MN
+								</p>
+							</div>
+						</a>
+					</article>
+					<article class="ctabox__cta" data-js-ctabox-item>
+						<a href="https://bitcointalk.org/index.php?topic=421615.0" target="_blank" class="ctabox__link">
+							<div class="ctabox__imagewrapper">
+								<img src="/assets/img/community/bitcointalk_logo.png" class="ctabox__image" >
+							</div>
+							<div class="ctabox__text">
+								<h4 class="ctabox__title">MONITORING</h4>
+								<p>
+									Budget
+									<br>
+									Voting Monitoring
+								</p>
+							</div>
+						</a>
+					</article>
+					<article class="ctabox__cta" data-js-ctabox-item>
+						<a href="https://forum.bitcoin.com/dash/" target="_blank" class="ctabox__link">
+							<div class="ctabox__imagewrapper">
+								<img src="/assets/img/community/bitcoindotcom_logo.png" class="ctabox__image" >
+							</div>
+							<div class="ctabox__text">
+								<h4 class="ctabox__title">VOTING STATUS</h4>
+								<p></p>
+							</div>
+						</a>
+					</article>
+				</div>
+			</div>
+		</div>
+	</section>
+	<section class="">
+		<div class="content">
+			<h2 class="section__title">{% t pages.budget.voting-header %}</h2>
+			<div class="row">
+				<div class="col-sm-6">
+					<h2>{% t pages.budget.voting-automatic %}</h2>
+					<p>{% t pages.budget.voting-automatic-text %}</p>
+					<a href="/network" class="btn-blue">{% t pages.budget.voting-guide-btn %}</a>
+					<a href="/network" class="btn-blue">{% t pages.budget.voting-manual-btn %}</a>
+					<a href="/network" class="btn-blue">{% t pages.budget.voting-central-btn %}</a>
+				</div>
+				<div class="col-sm-6">
+					<h2>{% t pages.budget.voting-tools %}</h2>
+					<p>{% t pages.budget.voting-tools-text %}</p>
+					<a href="">{% t pages.budget.decentralized-governance-link %}</a>
+					<a href="">{% t pages.budget.masternode-budget-link %}</a>
+					<a href="">{% t pages.budget.using-link %}</a>
+					<a href="">{% t pages.budget.dgb-link %}</a>
+					<a href="">{% t pages.budget.budget-finalization-link %}</a>
+				</div>
+			</div>
+		</div>
+	</section>
+
+	<section class="section section--currency" id="section-blockchain">
+		<h2 class="section__title">{% t pages.budget.exchange-heading %}</h2>
+		<table id="blocks">
+			<thead>
+				<tr>
+					<th>Height</th>
+					<th>Block Time</th>
+					<th>Transactions</th>
+					<th>Value Out</th>
+					<th>Difficulty</th>
+				</tr>
+			</thead>
+			<tbody>
+
+			</tbody>
+		</table>
+	</section>
+</div>
+<script type="text/javascript">

```

Comment:
We should probably move all javascript to js files and initialize through data-attributes (e.g. `<div clas="…" data-js-budget>`)